junebug/plugins/nginx/plugin.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import subprocess
+from os import path, remove
+from distutils.dir_util import mkpath
+from urlparse import urljoin
+
+from confmodel import Config
+from confmodel.fields import ConfigText
+
+from junebug.plugin import JunebugPlugin
+from junebug.utils import channel_public_http_properties
+
+
+log = logging.getLogger(__name__)
+DIRNAME = path.dirname(__file__,)
+
+
+class NginxPluginConfig(Config):
+    '''Config for :class:`NginxJunebugPlugin`'''
+
+    vhost_file = ConfigText(
+        "The file to write the junebug nginx vhost file to",
+        default='/etc/nginx/sites-enabled/junebug.conf', static=True)
+
+    locations_dir = ConfigText(
+        "The directory to write location block config files to",
+        default='/etc/nginx/includes/junebug/', static=True)
```

Comment:
@hodgestar I think we'll get a:
```
distutils.errors.DistutilsFileError: could not create 'foo/bar': Permission denied
```

Maybe that is okay? Not sure there's much that can be done about it.